### PR TITLE
[ticket/13558] Add smtp SSL context configuration options

### PR DESCRIPTION
**Pull request by rxu**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

<a href="https://tracker.phpbb.com/browse/PHPBB3-13558">PHPBB3-13558</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set WIP

**Comment by rxu** [comment]
!unset WIP

File: phpBB/includes/acp/acp_board.php [hunk 1]
```diff
@@ -454,6 +454,9 @@ function main($id, $mode)
 						'smtp_auth_method'		=> array('lang' => 'SMTP_AUTH_METHOD',		'validate' => 'string',	'type' => 'select', 'method' => 'mail_auth_select', 'explain' => true),
 						'smtp_username'			=> array('lang' => 'SMTP_USERNAME',			'validate' => 'string',	'type' => 'text:25:255', 'explain' => true),
 						'smtp_password'			=> array('lang' => 'SMTP_PASSWORD',			'validate' => 'string',	'type' => 'password:25:255', 'explain' => true),
+						'ssl_verify_peer'		=> array('lang' => 'SSL_VERIFY_PEER',		'validate' => 'bool',	'type' => 'radio:yes_no', 'explain' => true),
```
Comment (Nicofuma) [inline phpBB/includes/acp/acp_board.php]: it should be prefixed by smtp_

**Comment by Nicofuma** [comment]
Please add a note in the description saying it is for advanced use only as it can have security implications.

**Comment by rxu** [comment]
@Nicofuma Should I add a note to the header description or create a special one for those 3 settings?

**Comment by Nicofuma** [comment]
Looks good like that, thanks

**Pull request merged**